src/ontology/uberon-edit.obo

```diff
@@ -22982,6 +22982,7 @@ relationship: mutually_spatially_disjoint_with UBERON:0001630 ! muscle organ
 relationship: mutually_spatially_disjoint_with UBERON:0002385 ! muscle tissue
 relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
 relationship: part_of UBERON:0001434 ! skeletal system
+relationship: RO:0000052 UBERON:0008789 ! inheres in cranial fossa
```

Comment:
@megbalk 
The QC failed with this message:
ERROR: Undeclared relation: RO:0000052
RO:0000052 is 'characteristic of'...? Not sure what's going on there, but that's where you might want to look :-)
Thank you.

Reply:
@paolaroncaglia thank you!

Reply:
I think I found it in the antorbital term. I removed it to see if it solves the issue. If need be I can change it to "part of" some "cranial fossa" as it does occur in there.